examples/interop-examples/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "interop-examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "http-server -c-1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "ipfs-daemon": "^0.3.0-beta.16"
```

Comment:
When `ipfs-daemon` is out of the equation, it will be so much faster to get this example to work because one less npm install of the whole thing is needed.